nautobot/dcim/lookups.py

```diff
@@ -1,9 +1,33 @@
-from django.contrib.postgres.fields.array import ArrayContains
+from django.db import NotSupportedError
+from django.db.models.lookups import Lookup, Contains
 
 from nautobot.dcim.utils import object_to_path_node
 
 
-class PathContains(ArrayContains):
+class PathContains(Lookup):
+    lookup_name = "pathcontains"
```

Comment:
Seems to me leaving the `lookup_name = "contains"` would maintain the same external interface as before (e.g. `path__contains`). Is there a deliberate reason you renamed this lookup?

Reply:
The semantics of this lookup are changing from a SQL operator to a function call and I thought it was appropriate to reflect that in the lookup name. This lookup doesn't extend the `contains` lookup, it replaces it.

Reply:
I think that's fine that it replaces it TBH, since it's still performing the same semantic function.